File: go/rna-transcription/rna_transcription.go
package strand

import "strings"

func ToRNA(dna string) string {
  rna := ""
  rnaMap := map[string]string{
    "G": "C",
    "C": "G",
    "T": "A",
    "A": "U",
  }

  for _, nucleotide := range strings.Split(dna, "") {
    rna += rnaMap[nucleotide]
  }

  return rna
}

File: python/rna-transcription/rna_transcription.py
DNA_to_RNA = {
    'G': 'C',
    'C': 'G',
    'T': 'A',
    'A': 'U'
}

def to_rna(dna_strand):
    return "".join([DNA_to_RNA[nucleotide] for nucleotide in dna_strand])
